packages/nextjs/app/builders/0x5E71Cd371cD497A164da8f97aeb853efD160F5f5/page.tsx

```diff
@@ -0,0 +1,127 @@
+            style={{ borderColor: "#8246E5" }} // Update border color
+          />
+          <h1 className="text-4xl font-bold mt-4 text-center">Aniket Chaudhary</h1>
+          <p className="text-xl text-gray-600 mt-2 text-center">Blockchain and Web3 Developer</p>
+        </div>
+      </header>
+
+      <main className="max-w-4xl mx-auto mt-10 space-y-8">
+        <section className="bg-white p-6 shadow-md rounded-lg">
+          <h2 className="text-3xl font-semibold mb-4" style={{ color: "#8246E5" }}>
+            Introduction
+          </h2>
+          <p>
+            I am a final year B.Tech-IT student from India, passionate about building the future with blockchain and web
+            development. I am particularly excited about creating decentralized applications (dApps) and exploring the
+            cutting-edge technologies that drive innovation in this space.
+          </p>
+        </section>
+
+        <section className="bg-white p-6 shadow-md rounded-lg">
+          <h2 className="text-3xl font-semibold mb-4" style={{ color: "#8246E5" }}>
+            Education
+          </h2>
+          <ul className="list-disc list-inside ml-4">
+            <li>Bachelor of Technology in Information Technology</li>
+            <li>Courses in Blockchain and Web3 Security (Cyfrin Updraft)</li>
+            <li>Courses in Web3 Development (Coursera, Udemy and Youtube) </li>
+          </ul>
+        </section>
+
+        <section className="bg-white p-6 shadow-md rounded-lg">
+          <h2 className="text-3xl font-semibold mb-4" style={{ color: "#8246E5" }}>
+            Skills and Expertise
+          </h2>
+          <ul className="list-disc list-inside ml-4">
+            <li>Smart Contracts Development (Solidity)</li>
+            <li>TypeScript, JavaScript</li>
+            <li>Decentralized Applications (DApps)</li>
+          </ul>
+        </section>
+
+        <section className="bg-white p-6 shadow-md rounded-lg">
+          <h2 className="text-3xl font-semibold mb-4" style={{ color: "#8246E5" }}>
+            Projects
+          </h2>
+          <ul className="list-disc list-inside ml-4">
+            <li>
+              <strong>SecureVote:</strong> A decentralized voting application with Aadhaar authentication, providing
+              secure and transparent voting solutions.
+            </li>
+            <li>
+              <strong>BUY-me-chai:</strong> A platform for donating cryptocurrency to content creators.
+            </li>
+            <li>
+              <strong>DEX(SpeedRun Ethereum):</strong> A decentralized exchange for trading ERC20 tokens and ETH.
+            </li>
+          </ul>
+        </section>
+
+        <section className="bg-white p-6 shadow-md rounded-lg">
+          <h2 className="text-3xl font-semibold mb-4" style={{ color: "#8246E5" }}>
+            Contact Me
+          </h2>
+          <p>Feel free to reach out:</p>
+          <ul>
+            <li>
+              Email:{" "}
```

Comment:
You might consider adding icons here to make it more appealing for the user. Instead of having Email, LinkedIn, etc.

Reply:
I have added the icons for each of the contact links : 
![Screenshot 2024-07-27 174403](https://github.com/user-attachments/assets/0a4f81f1-82cd-49f0-a997-7abb64b8876f)